utils.py

```
@@ -1,139 +1,167 @@
-
-
+import torch
+import pickle
+from pathlib import Path
+import os 
+from Bio.PDB.MMCIFParser import MMCIFParser
+import gzip
+from Bio.SeqUtils import seq1
+import obonet
+import numpy as np
+import argparse
```

Ruff (0.8.2)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)